src/page/ShippingStep.page.ts

```diff
@@ -0,0 +1,18 @@
+import { $, ElementFinder, browser } from 'protractor';
+
+export class ShippingStepPage {
+
+  private get proceedToCheckoutButton(): ElementFinder {
+    return $('#form > p > button > span');
+  }
+
+  private get termsCheckBox(): ElementFinder {
+    return $('#cgv');
+  }
+
+  public async goToPaymentStepPage() {
+    await this.termsCheckBox.click();
+    await (browser.sleep(3000));
+    return this.proceedToCheckoutButton.click();
```

Comment:
cambia esta linea por await this.proceedToCheckoutButton.click();

Reply:
Listo :)